src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,65 @@
-export const Pagination = () => {};
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handlePageChange = (page: number) => {
```

Comment:
you should reset link page number as well.

https://github.com/mate-academy/react_pagination/assets/117587463/95d87707-6eed-48c1-bdca-54a7b9063e05